.github/workflows/release-ci.yaml

```diff
@@ -14,3 +14,123 @@ jobs:
       - run: |
           echo "Event: ${{ github.event_name }}" >> "$GITHUB_STEP_SUMMARY"
           echo "Ref: ${{ github.ref_name }}" >> "$GITHUB_STEP_SUMMARY"
+
+  build:
+    name: Build
+    uses: ./.github/workflows/build.yaml
```

Comment:
We need the new roxctl binary later in the process. Since we need to build the binaries and images anyway, we do this by workflow call from the release workflow. Thereby, we can re-use the roxctl binary artifact that is uploaded in the build workflow.

Reply:
The only solution I found for that is to use `workflow_run` trigger similar to https://stackoverflow.com/a/64733705. 
However, that would trigger the release workflow for all completed workflows and we'd need to add checks to ensure we only run the release workflow only for actual releases...